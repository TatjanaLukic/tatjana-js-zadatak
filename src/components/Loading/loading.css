@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400&display=swap");
.loading {
  background-color: #8e0a8e;
  opacity: 0.3;
  width: 120px;
  padding: 5rem;
  margin-top: 6rem;
  margin-left: auto;
  margin-right: auto;
}

/*# sourceMappingURL=loading.css.map */
