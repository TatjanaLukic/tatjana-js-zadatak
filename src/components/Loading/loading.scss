@import "../../assets/variables";
.loading {
  background-color: $nav-second-color;
  width: 120px;
  padding: 5rem;
  margin-top: 6rem;
  margin-left: auto;
  margin-right: auto;
}
