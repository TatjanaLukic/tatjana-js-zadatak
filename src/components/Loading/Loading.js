import React from "react";
import "./loading.css";
import { ReactComponent as Icon } from "../../assets/images/circles.svg";

const Loading = () => (
  <div className="loading" data-testid="loading">
    <Icon />
  </div>
);

export default Loading;
