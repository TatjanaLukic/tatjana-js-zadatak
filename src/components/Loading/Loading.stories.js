import React from "react";
import Loading from "./Loading";

export default {
  title: "App/Loading",
  component: Loading,
};

const Template = () => <Loading />;

export const Default = Template.bind({});
