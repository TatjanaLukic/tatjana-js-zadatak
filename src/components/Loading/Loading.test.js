import React from "react";
import { render } from "@testing-library/react";
import Loading from "../Loading/Loading";

describe("<Loading />", () => {
  it("spanshot matches", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
