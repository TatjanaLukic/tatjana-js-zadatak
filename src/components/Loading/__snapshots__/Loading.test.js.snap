// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> spanshot matches 1`] = `
<div>
  <div
    class="loading"
    data-testid="loading"
  >
    <svg>
      circles.svg
    </svg>
  </div>
</div>
`;
